### ISSUE TITLE
Pick a new string to be used for PAYLOADFORMAT, and support it properly

### ISSUE THREAD

**Issue by dralley** [rpm-software-management/rpm#2447]
RPMs produced since 4.12 use a [stripped down payload format](https://github.com/rpm-software-management/rpm/pull/2433) based on CPIO, but not compatible with CPIO.

However this isn't currently reflected by the handling of the `PAYLOADFORMAT` tag, which always writes `cpio`.

https://github.com/rpm-software-management/rpm/blob/97b75f592d7731aa3105f47b2f5b8a62a2e1ce12/build/pack.c#L296

This new value (whatever it is) should also be added to the support checking logic:

https://github.com/rpm-software-management/rpm/blob/97b75f592d7731aa3105f47b2f5b8a62a2e1ce12/build/pack.c#L296

**Comment by pmatilai** [comment]
Yep, it's unfortunate that the stripped cpio format wasn't handled back then. Adding a new string for it now has a good chance of breaking something.

**Comment by pmatilai** [comment]
Not just a chance to break, but would cause a complete backwards compatibility break for what is only a cosmetical benefit. That is a no go.

We could use recycle the tag to a new id, but whether this is really worth it I don't know. A new payload format would have to be tracked by a new rpmlib() dependency, and that part was actually covered when the new format was introduced.

**Comment by dralley** [comment]
@pmatilai Shouldn't this be tagged v6?

**Comment by pmatilai** [comment]
No, this is not happening on v6 because of the massive backwards compatibility break it would cause, for very close to zero benefit. Dropped it from the project too.
